docs/workflow/testing/mono/testing.md

```diff
@@ -1,54 +1,83 @@
 # Running Tests using Mono Runtime
 
+Before running any tests, build mono in the way that you would like to test with.
+See the instructions for [Building Mono](../../building/mono/README.md)
+
 ## Running Runtime Tests
-We currently only support running tests against coreclr.  There are additional mono runtime tests in mono/mono, but they
-have not been moved over yet. Simply run the following command:
+* Build and run runtime tests against Mono JIT:
```

Comment:
```suggestion
### Desktop:
```

Reply:
There will be instructions for other flavors of desktop Mono, for example, AOT, which will be slightly different. I would prefer to be more specific here.